common/enablers/epochFlags.go

```diff
@@ -103,6 +103,7 @@ type epochFlagsHolder struct {
 	fixDelegationChangeOwnerOnAccountFlag       *atomic.Flag
 	dynamicGasCostForDataTrieStorageLoadFlag    *atomic.Flag
 	nftStopCreateFlag                           *atomic.Flag
+	changeOwnerAddressCrossShardThroughSCFlag   *atomic.Flag
```

Comment:
missing tests in config/tomlConfig_test.go ?

Reply:
done